### PR TITLE
Map for projects

### PR DESCRIPTION
**Pull request by ghost**
Map of leaflet and button working with fake coordinates

### CODE REVIEW

File: src/home/Home.jsx [hunk 1]
```diff
@@ -251,18 +258,20 @@ function Home() {
         <Row className="resultsNav w-100">
           <Col xs={12} md={6} className="mb-3">
             <Button className="bigNavBtn" title="Add a new project">
-              Project <PlusSquareFill />
+              <a
+                href="https://boninabox.geobon.org/backend/submit-project"
+                target="_blank"
+              >
```
Comment (erikasv) [inline src/home/Home.jsx:264]: add `rel="noopener noreferrer"`

File: src/home/LeafletMap.jsx [hunk 2]
```diff
@@ -0,0 +1,46 @@
+import React, { useState, useEffect } from "react";
+import L from "leaflet";
+import "leaflet/dist/leaflet.css";
+import "font-awesome/css/font-awesome.min.css"; // Import Font Awesome CSS
```
Comment (erikasv) [inline src/home/LeafletMap.jsx:4]: remove comment

File: src/home/LeafletMap.jsx [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+  useEffect(() => {
+    if (visible) {
+      const mapInstance = L.map("leaflet-map").setView([4.5709, -74.2973], 6);
+
+      L.tileLayer("https://{s}.tile.openstreetmap.org/{z}/{x}/{y}.png", {
+        attribution:
+          '&copy; <a href="https://www.openstreetmap.org/copyright">OpenStreetMap</a> contributors',
+      }).addTo(mapInstance);
+
+      // Create a custom icon for each marker
```
Comment (erikasv) [inline src/home/LeafletMap.jsx:18]: remove commnet

File: src/home/Home.jsx (hunk 1 above)
Comment (ManuelStardust) [reply src/home/Home.jsx:265]: Done

File: src/home/LeafletMap.jsx (hunk 2 above)
Comment (ManuelStardust) [reply src/home/LeafletMap.jsx]: Done

File: src/home/LeafletMap.jsx (hunk 3 above)
Comment (ManuelStardust) [reply src/home/LeafletMap.jsx]: Done

File: src/home/LeafletMap.jsx
[the remark this answers is not in the document]
Comment (ManuelStardust) [reply src/home/LeafletMap.jsx:1]: Done

**Review by erikasv — APPROVED** [review]

**Pull request merged**